src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/NormalizedMultiValuesTest.java

```diff
@@ -164,6 +243,29 @@ public void testGetAllNormalizedMultiValuesWfs20() throws Exception {
         checkStation2Gml32(document);
     }
 
+    @Test
+    public void testGetAllNormalizedMultiValuesWfsJson20() throws Exception {
+        // check if this is an online test with a JDBC based data store
+        if (notJdbcBased()) {
```

Comment:
Why wouldn't this work outside of a database scenario?

Reply:
This is due 1..n cardinality on app-schema is only supported on JDBC and Solr backends, on Geoserver documentation: "Mappings with a one to many relationship are supported with a custom syntax in JDBC based data stores and Apache Solr data store."
https://docs.geoserver.org/stable/en/user/data/app-schema/mapping-file.html#attributes-with-cardinality-1-n

Reply:
I see, good to go then.